### ISSUE TITLE
several whitespaces at a stretch

### ISSUE THREAD

**Issue by burrbull** [oli-obk/rust-si#5]
panicked with
thread '<main>' panicked at 'called `Result::unwrap()` on an `Err` value: ParseFloatError { kind: Empty }', src/libcore/result.rs:736
if more than 1 whitespace after value in parsed string (for example 2 spaces or space + Enter)

**Comment by oli-obk** [comment]
thanks, I'll look into it asap

**Comment by oli-obk** [comment]
I can't really reproduce it. Do you have a code example? are you using nightly or stable?

**Comment by burrbull** [comment]
Sorry. I had to explain in more detail.
It's happen when I read some values together. For example a vector:
```rust
let mut a: Vec<f64> = Vec::new();
for _ in 0..n {
	let f: f64 = read!();
	a.push(f);
```
If these values are separated with 1 space => OK.
But if I use 2 ore more whitespaces between values => Error.
Ubuntu. Rust 1.4 and nightly 1.6 (build 2015.11.10)

**Comment by oli-obk** [comment]
This is fixed in 0.1.6
